mizar/networkpolicy/networkpolicy_util.py

```diff
@@ -350,12 +350,12 @@ def fill_cidrs_from_directional_traffic_rules(self, data, policy_name, indexed_p
                             "pod_label_map": self.get_pod_label_values_by_label_dict(rule_item["podSelector"]["matchLabels"]),
                             "namespace_label_map": self.get_namespace_label_values_by_label_dict(rule_item["namespaceSelector"]["matchLabels"])
                         }
-
-                    namespaces = kube_list_namespaces_by_labels(networkpolicy_opr.core_api, rule_item["namespaceSelector"]["matchLabels"])
-                    if namespaces is not None:
-                        namespace_set = set()
-                        for namespace in namespaces.items:
-                            namespace_set.add(namespace.metadata.name)
+                    else:
```

Comment:
this else statement corresponding to if direction == "ingress", which is direction == "egress" right? ingress and egress rules should be similar. Basically please clarify the logic. Please add comments in network policy util.

Reply:
For ingress traffic, use label based policy. The label based policy only fits for ingress for now. For egress traffic, still keep original usage which is ip based policy. 
Added comments in the code to tell the difference.